packages/cli/src/services/workflow-statistics.service.ts

```diff
@@ -90,7 +109,7 @@ export class WorkflowStatisticsService extends EventEmitter {
 		const owner = await this.ownershipService.getProjectOwnerCached(project.id);
 
 		let metrics = {
-			user_id: owner?.id,
+			user_id: owner!.id,
```

Comment:
No context here but `owner` looks like a project, not a user.

Reply:
@valya Is this a bug?

Reply:
`owner` is a `User` 
![image](https://github.com/user-attachments/assets/9e32bd77-f6d5-483d-ba49-39a6cc156746)